### DISCUSSION TITLE
ValueError: non-hexadecimal number found in fromhex() arg at position 97

### DISCUSSION THREAD

**Discussion by heeroz** [Chia-Network/chia-blockchain#4092] [Support]
I want to plot on a Harvester machine（Windows 10）. I got farm public and pool public from the full node.
After I installed chia(1.1.4) on Harvester machine and execute the "chia plot create" command, I got following errors.

`PS C:\Users\mining\AppData\Local\chia-blockchain\app-1.1.4\resources\app.asar.unpacked\daemon> .\chia plots create -f <farmer-public-key> -p <pool-public-key> -k 32 -t D:\plot\  -d E:\farm\`
`Traceback (most recent call last):`
`  File "chia\cmds\chia.py", line 81, in <module>`
`  File "chia\cmds\chia.py", line 77, in main`
`  File "click\core.py", line 829, in __call__`
`  File "click\core.py", line 782, in main`
`  File "click\core.py", line 1259, in invoke`
`  File "click\core.py", line 1259, in invoke`
`  File "click\core.py", line 1066, in invoke`
`  File "click\core.py", line 610, in invoke`
`  File "click\decorators.py", line 21, in new_func`
`  File "chia\cmds\plots.py", line 135, in create_cmd`
`  File "chia\plotting\create_plots.py", line 64, in create_plots`
`ValueError: non-hexadecimal number found in fromhex() arg at position 97`
`[4620] Failed to execute script chia`

Any advice will be help.
Thank you in advance.

**Comment by vulovicv23** [comment]
Same here, any updates on this?

**Reply by heeroz** [reply]
I copied full node ca dir(C:\Users\<username>\.chia\mainnet\config\ssl\ca) and seed text to the harvester machine.
Use following commands to initiation on the harvester machine.

```
chia.exe keys add -f seed.txt
chia.exe init -c <ca dir>
chia.exe start harvester
```

And then I can plot without keys.

**Comment by ghost** [comment]
It seems you wrote the wrong Pool public key.

Use following command to check.
`chia.exe keys show`

Don't added "0x" before the key. (hpool's plotter software requires this)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
